### DISCUSSION TITLE
0.16.1 release?

### DISCUSSION THREAD

**Discussion by raybellwaves** [anitagraser/movingpandas#323] [General]
I'm looking to release now https://github.com/movingpandas/movingpandas/pull/315 is merged.

I had some previous testing on a project which failed once the timezone was being stripped via https://github.com/movingpandas/movingpandas/commit/53c6385e8239edd280d5e40eb9a6dbccdf21207d

Anything else desirable which should be added for a release? Or most importantly are there any bugs for why we should release?

I'm happy to do the release

**Comment by anitagraser** [comment]
Let's also merge https://github.com/movingpandas/movingpandas/pull/316 and release 0.16.1

**Reply by raybellwaves** [reply]
Am I ok to merge https://github.com/movingpandas/movingpandas/pull/316 and do the release?

**Reply by anitagraser** [reply]
Yes

**Reply by anitagraser** [reply]
I've merged the pr.

**Reply by anitagraser** [reply]
Hi @raybellwaves. Are you already working on the release or do you want me to jump in?

**Reply by raybellwaves** [reply]
sorry been off recently. If you're happy to do so that would be great thanks

**Reply by anitagraser** [reply]
👍 done

**Reply by raybellwaves** [reply]
Thank you very much
